src/main/resources/templates/layouts/main.html

```diff
@@ -123,15 +96,9 @@
 		<script th:src="@{/webjars/bootstrap/js/bootstrap.min.js}"></script>
 		<div layout:fragment="javascript">
 		</div>
-		<script type="text/javascript">
-		//<!--
-		$(function() {
-				$('#sign-out').click(function(e) {
-						e.preventDefault();
-						$('#sign-out-form').submit();
-				});
-		});
-		//-->
-		</script>
+    	<script th:src="@{/layouts/main.js}" type="text/javascript"></script>
+    	<script type="text/javascript">
+			mainLayout.initialize();
```

Comment:
Any chance we can externalize this too? I'd like to externalize everything so we can setup a Content Security Policy http://www.html5rocks.com/en/tutorials/security/content-security-policy/